app/scripts/lib/multichain-api/scope/supported.ts

```diff
@@ -8,20 +8,17 @@ import {
 } from '@metamask/utils';
 import { toHex } from '@metamask/controller-utils';
 import { InternalAccount } from '@metamask/keyring-api';
-import MetaMaskOpenRPCDocument from '@metamask/api-specs';
 import { isEqualCaseInsensitive } from '../../../../../shared/modules/string-utils';
-import { KnownCaipNamespace } from './scope';
-
-export const validRpcMethods = MetaMaskOpenRPCDocument.methods.map(
-  ({ name }) => name,
-);
-
-// TODO: remove invalid notifications
-export const validNotifications = [
-  'accountsChanged',
-  'chainChanged',
-  'eth_subscription',
-];
+import {
+  KnownCaipNamespace,
+  KnownNotifications,
+  KnownRpcMethods,
+  KnownWalletNamespaceRpcMethods,
+  KnownWalletRpcMethods,
+  NonWalletKnownCaipNamespace,
```

Comment:
I wonder if we can/should remove `Known` from these. Perhaps it can be considered implied?

Reply:
The reason the prefix exists on `KnownCaipNamespace` is because of a name collision with the actual `CaipNamespace` type otherwise https://github.com/MetaMask/utils/blob/main/src/caip-types.ts#L29

We can remove these from the other constants though. Would that work for you?

Reply:
🤔 no I think maybe we should leave these types with the `Known` prefix for a bit of additional context